<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AllCarsWiewForm</class>
 <widget class="QWidget" name="AllCarsWiewForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>338</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="ActionPushButton" name="pushButtonAdd">
         <property name="text">
          <string>Добавить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="ActionPushButton" name="pushButtonDel">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="TableWiewCars" name="tableView"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TableWiewCars</class>
   <extends>QTableView</extends>
   <header>tablewiewcars.h</header>
  </customwidget>
  <customwidget>
   <class>ActionPushButton</class>
   <extends>QPushButton</extends>
   <header>actionpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonAdd</sender>
   <signal>clicked()</signal>
   <receiver>AllCarsWiewForm</receiver>
   <slot>slotPushButtonAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>50</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonDel</sender>
   <signal>clicked()</signal>
   <receiver>AllCarsWiewForm</receiver>
   <slot>slotPushButtonDel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotPushButtonAdd()</slot>
  <slot>slotPushButtonDel()</slot>
 </slots>
</ui>
